pandas/tests/test_generic.py

```diff
@@ -996,6 +996,59 @@ def test_describe_percentiles_insert_median(self):
         self.assertTrue('0%' in d1.index)
         self.assertTrue('100%' in d2.index)
 
+    def test_describe_percentiles_unique(self):
+        # GH13104
+        df = tm.makeDataFrame()
+        with self.assertRaises(ValueError):
+            df.describe(percentiles=[0.1, 0.2, 0.4, 0.5, 0.2, 0.6])
+        with self.assertRaises(ValueError):
+            df.describe(percentiles=[0.1, 0.2, 0.4, 0.2, 0.6])
+
+    def test_describe_percentiles_formatting(self):
+        # GH13104
```

Comment:
No, there's only one for an empty series, which is different. `pd.concat` raises on empty frame. I can fix it. But do we want an exception or the same output as for series? (For series it's a column with count = 0 and NaN's for other numeric stats.)